src/main/java/uk/gov/hmcts/reform/ccd/client/model/SearchCriteria.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.ccd.client.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.Optional;
+
+public class SearchCriteria {
+    private static final Integer MINIMUM_SIZE_PER_PAGES = 10;
+
+    private final ObjectMapper mapper;
+
+    public SearchCriteria(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
+
+    public String matchAllQuery(Integer size) {
+        JsonNode query = mapper.createObjectNode()
+            .put("size", Optional.ofNullable(size).orElse(MINIMUM_SIZE_PER_PAGES))
```

Comment:
Suggestion:
`The above forces the caller to pass either null or some value. which is not very user friendly.`

we can have overloaded methods like:
```
public String matchAllQuery(Integer size) {
...
.put("size",  size)
...
```
and other one:

```
public String matchAllQuery() {
return matchAllQuery(MINIMUM_SIZE_PER_PAGES)
}
```

Reply:
Thank you very much @dharmendrak  👍